languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -34110,6 +34124,83 @@ though, and has since been slightly modified)-->
+                         </unify-ignore>
+                         <token postag="V.*" postag_regexp="yes"/>
+                         <unify-ignore>
+                             <token postag="R pers suj.*" postag_regexp="yes" regexp="yes">(?-i)[a-z].*</token>
+                         </unify-ignore>
+                         <token postag="V.*" postag_regexp="yes"/>
+                     </marker>
+                 </unify>
+             </pattern>
+             <message>Une virgule semble être requise dans ce contexte.</message>
+             <suggestion>\1 \2, \3 \4</suggestion>
+             <example correction="Nous montons, nous descendons"><marker>Nous montons nous descendons</marker> nous ne savons plus où aller.</example>
+         </rule>
+         <rule>
+             <antipattern>
+                 <token postag="R pers suj.*" postag_regexp="yes"/>
+                 <token inflected="yes">prendre</token>
+                 <token postag="V.*" postag_regexp="yes"/>
+                 <token>et</token>
+                 <token postag="V.*" postag_regexp="yes"/>
+             </antipattern>
+             <pattern>
+                 <unify>
+                     <feature id="person"/>
+                     <feature id="tense"/>
+                         <unify-ignore>
+                             <token postag="R pers suj.*" postag_regexp="yes"/>
+                         </unify-ignore>
+                     <marker>
+                         <token postag="V.*" postag_regexp="yes">
+                             <exception postag="V etre.*" postag_regexp="yes"/>
+                             <exception postag="V avoir.*" postag_regexp="yes"/>
+                         </token>
+                         <token postag="V.*" postag_regexp="yes" regexp="yes">(?-i)[a-z].*<exception regexp="yes">casques?|confiance</exception></token>
+                     </marker>
+                     <unify-ignore>
+                         <token>et</token>
+                     </unify-ignore>
+                     <token postag="V.*" postag_regexp="yes"/>
+                 </unify>
+             </pattern>
+             <message>Une virgule semble être requise dans ce contexte.</message>
+             <suggestion>\2, \3</suggestion>
+             <example correction="monte, descend">Elle <marker>monte descend</marker> et recule.</example>
+             <example>Elle <marker>prend place</marker> et trouve une feuille.</example>
+             <example>Il <marker>aime Sue</marker> et accepte tout pour elle.</example>
+             <example>Il <marker>reprend confiance</marker> et retourne au combat.</example>
+         </rule>
+         <rule>
+             <pattern>
+                 <unify>
+                     <feature id="person"/>
+                     <feature id="tense"/>
+                 <unify-ignore>
+                             <token postag="R pers suj.*" postag_regexp="yes"/>
+                 </unify-ignore>
```

Comment:
By the way, in this case, you could place the `R pers` outside of the `<unify>`, right after `<pattern>`... that'll save you the `<unify-ignore>` in this case.

Reply:
Indeed! It is working! That's great!